sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/StandardMetricsExtractionProcessor.cs

```diff
@@ -71,6 +82,31 @@ private void ReportRequestDurationMetric(Activity activity, string statusCodeAtt
             _requestDuration.Record(activity.Duration.TotalMilliseconds, tags);
         }
 
+        private void ReportDependencyDurationMetric(Activity activity)
+        {
+            var monitorTags = TraceHelper.EnumerateActivityTags(activity);
+
+            var dependencyTarget = monitorTags.MappedTags.GetDependencyTarget(monitorTags.activityType);
```

Comment:
In a follow up PR we might need to reconsider combining `GetDependencyTarget` and `GetDependencyType` and return `Dependency` objects with parameter `Target` and `Type`.

Reply:
tracking all follow ups here https://github.com/Azure/azure-sdk-for-net/issues/34012